samples/server.c

```diff
@@ -227,15 +227,15 @@ static void do_dma_io(vfu_ctx_t *vfu_ctx, struct server_data *server_data)
 
     memset(buf, 'A', count);
     get_md5sum(buf, count, md5sum1);
-    printf("%s: WRITE addr %p count %d\n", __func__,
+    vfu_log(vfu_ctx, LOG_DEBUG, "%s: WRITE addr %p count %d", __func__,
```

Comment:
This isn't called by the library so I think `printf` is fine.

Reply:
it's so we get the consistent prefix I added.